review-notifications/src/Options.js

```diff
@@ -9,13 +9,14 @@ export const initialState = {
   auth: false,
   token: '',
   newRepo: '',
-  reposLinks: [],
   prTypes: [
     { id: 1, value: 'Created', isChecked: false },
     { id: 2, value: 'Assigned', isChecked: false },
     { id: 3, value: 'Mentioned', isChecked: false },
     { id: 4, value: 'Review request', isChecked: false },
+    { id: 5, value: 'From followed repositories', isChecked: false },
```

Comment:
Export all possible strings that are used in more than one place to `enum`/`const`